source/_patterns/02-components/overflow-menu/overflow-menu.scss

```diff
@@ -13,15 +13,21 @@
 	}
 
 	summary {
-		@include icon(
-			$cmp-overflow-menu-icon,
-			20,
-			"outline",
-			$partial: $partial
-		);
-
 		display: inline-block;
 
+		&:not([data-icon]) {
+			@include icon(
+				$cmp-overflow-menu-icon,
+				20,
+				"outline",
+				$partial: $partial
+			);
+		}
+		// Possibility to define an individual icon
+		&[data-icon] {
+			@include icon("", 20, "outline", $partial: $partial);
```

Comment:
ist it also possible to include a solid icon then?

Reply:
good question.